sensor_msgs/msg/CompressedImage.msg

```diff
@@ -38,4 +41,10 @@ string format                # Specifies the format of the data
                              #   need for successful decoding of the image. Refer to
                              #   documentation of the other transports for details.
 
+string compression_type      # Compression type used (jpeg, png, theora, etc)
+
+uint64 sequence_number       # sequence number
```

Comment:
I don't think I agree that this should have a sequence number.  A compressed image doesn't logically have one of these, and we don't have this in Image.msg, for instance.